test/render/test-utils/get-model.js

```diff
@@ -68,8 +68,8 @@ void main(void) {
 }
 `,
       geometry: new Geometry({
-        attributes: normalizeAttributes(attributes),
-        drawMode: attributes.indices ? gl.TRIANGLES : gl.POINTS
+        attributes: normalizeAttributes(data),
```

Comment:
Foods for thought: why do they need to be normalized when they have just gone through a rigorous conformance test? Shouldn't they already be "normal"?

Perhaps it is about mapping GLTF attribute names to match luma.gl heuristics? Since we are adding glTF support we should of course make sure the luma.gl heuristics understand glTF attribute names out of the box, perhaps if we do that this is not necessary?

Reply:
`normalizeAttributes` is basically mapping attributes from GLTF attribute names to the target model's attribute names.